etc/config.sample.toml

```diff
@@ -71,11 +71,15 @@
 
   # CacheMaxMemorySize is the maximum size a shard's cache can
   # reach before it starts rejecting writes.
-  # cache-max-memory-size = 1048576000
+  # Valid size suffixes are k, m, or g (case insensitive).
+  # Vaues without a size suffix are in bytes.
+  # cache-max-memory-size = "1g"
```

Comment:
Just a note for those who might check this, the sample configuration file has the wrong value. The code has 1 GB as the default and 1 GB in bytes is 1073741824 bytes, not 1048576000.

Reply:
I see, the old sample config value was 1000m but the `const DefaultCacheMaxMemorySize` is actually 1g (i.e. 1024m). Fixing the sample config to 1g makes sense here.